checkov/terraform/checks/graph_checks/azure/AzureMLWorkspacePublicNetwork.yaml

```diff
@@ -0,0 +1,16 @@
+metadata:
+  id: "CKV2_AZURE_49"
+  name: "Ensure that Azure Machine learning workspace is not configured with overly permissive network access"
+  category: "GENERAL_SECURITY"
```

Comment:
I think we have the `NETWORKING` category which mighe be better suited based on the definition of this check